teams/developer-relations/guide-to-being-a-manager/README.md

```diff
@@ -68,6 +152,43 @@ Every week, there are a couple essential things that every manager should take c
+
+Stemming from an effort to be more transparent and let developers be aware of how their work contributes to the organization as a whole, this list both ensures proper workflow and a more stylistically workflow. As more and more of the list is checked off, the developer is able to see their progress. Ideally, the list for a specific PR is completely checked off by the end of the week.
+
+## _Quality Assurance Procedures for Managers and Developers \[Jason\]_
+
+We want to the developers and the managers to provide give their input for each aspect of our projects. Therefore, we need procedures that allow the developers to quickly and effectively work together to complete tasks while maintaining a standard of quality.
+
+### Procedure:
+
+* [ ] Managers and Kevin conduct code reviews through GitHub
+  * [ ] Checking Issues and contributions
+    * Each time a developer makes a Pull Request, they need to provide a list of check boxes of items they addressed for their work that week.
+    * When managers review each Pull Request, they must check to ensure that all assigned issues in the milestone were completed and the checkbox contributions match the assigned issues.
+  * [ ] Every checkbox must be addressed with a comment by the reviewer
+    * A comment can acknowledge that the developer did something correctly or give constructive feedback.
+
+## _Revision by Head of Developer Relations and President \[Jeff\]_
+
+Here at Bit, we place heavy emphasis that all content and material developed by Bit is up to our standards. In addition to material being reviewed by respective managers, the material will undergo further review by the Head of Developer Relations, as well as the President of Bit Project. We have many developer teams at Bit and this process is necessary to ensure all material is synchronous and consistent.
+
+The Head of Developer Relations will occasionally audit a random Pull request from a developer to a manager and ensure proper feedback is being provided. While the Head is auditing, he will be looking to ensure the content matches Bit Project's long term goals and is consistent with the other material. He will refer to the pull request template mentioned above while performing his audit. This means ensuring all content fulfills the criteria specified by the pull request template which will result in more thorough content. 
+
+The president, will also occasionally audit the Head of Developer Relation's pull requests to ensure quality is consistent through all curriculum by Bit Project. It will conducted in a similiar process to Code Reviews performed by managers and the Head of Developer Relations.
+
+In order for curriculum to have a level of quality expected of Bit Project, we have implemented 3 checks. 
+
+ 1. Approval by Manager
+
+ 2. Approval by Head of Developer Relations
+
+ 3. Approval by President of Bit Project
+
```

Comment:
I feel like there should be a section just about maintaining good communication with your team members (through slack?) and the importance of keeping track of everyone's progress throughout the week